apisix/plugins/csrf.lua

```diff
@@ -129,6 +135,10 @@ function _M.access(conf, ctx)
         return
     end
 
+    if conf.expires == 0 then
+        return
+    end
```

Comment:
Ahh, sorry I confused you. I didn't mean to ignore csrf token validation. I meant we can ignore `token has expired check`.